app/routes/docs.sample.kafka.md

```diff
@@ -0,0 +1,121 @@
+---
+handle:
+  breadcrumb: Kafka Usage
```

Comment:
Can you think of a less vague title than "Usage"?

Reply:
Further down it says "Kafka Message Consumption".  Perhaps something like that would be more specific.

Reply:
How about "Consuming Kafka Messages?"